runtime/pack/dist/opt/nohlsearch/plugin/nohlsearch.vim

```diff
@@ -5,6 +5,12 @@
 " turn off hlsearch after:
 " - doing nothing for 'updatetime'
 " - getting into insert mode
```

Comment:
These comments seem to fit better just above the autocommands, not here.

Reply:
Actually that's not necessarily true, as they can serve as the description of the behavior of the file here.

Reply:
well :)